### ISSUE TITLE
newrelic_infra_alert_condition - add support for integration_provider

### ISSUE THREAD

**Issue by ConstantinElse** [terraform-providers/terraform-provider-newrelic#35]
### Terraform Version
terraform 0.11.7
newrelic provider 1.0.1

### Affected Resource(s)
- newrelic_infra_alert_condition

### Terraform Configuration Files
```hcl
resource "newrelic_infra_alert_condition" "elb_host_count" {
  policy_id = "${newrelic_alert_policy.alerting.id}"

  name       = "ELB Healthy Host Count"
  enabled    = true
  type       = "infra_metric"
  event      = "LoadBalancerSample"
  select     = "provider.healthyHostCount.Average"
  comparison = "below"
  where      = "(`ec2Tag_environment` = '${var.environment}')"

  critical {
    duration      = 10
    value         = 1
    time_function = "all"
  }
}
```

### Expected Behaviour
The result should be a `working` alert condition for ELB integration in NewRelic.

### Actual Behaviour
The created alert condition doesn't pick up and display the metric from ELB. The displayed graph is empty in contrast to a manually configured alert.

![image](https://user-images.githubusercontent.com/40232958/43220294-99bcc6e0-9041-11e8-9dfb-e3c86d252bf4.png)

### Important Factoids
Querying the REST API shows that there is a need for additional argument for AWS ELB integration called `integration_provider` which is missing (or not available now ).

```json
{
  "data": [
    {
      "type": "infra_metric",
      "name": "elb",
      "enabled": true,
      "filter": {
        "and": [
          {
            "is": {
              "provider.elbTag_environment": "prod"
            }
          }
        ]
      },
      "id": 6304632,
      "created_at_epoch_millis": 1532542213665,
      "updated_at_epoch_millis": 1532542213736,
      "policy_id": 287280,
      "event_type": "LoadBalancerSample",
      "select_value": "provider.healthyHostCount.Average",
      "comparison": "below",
      "critical_threshold": {
        "value": 1,
        "duration_minutes": 10,
        "time_function": "all"
      },
      "integration_provider": "Elb"
    },
```

**Comment by triggert** [comment]
Just ran into the same issue myself.

**Comment by ConstantinElse** [comment]
@triggert the problem is that the `go-newrelic` library, used by terraform provider doesn't support it too yet.

**Comment by triggert** [comment]
@ConstantinElse Thanks - that worked with the addition of updates to the resource_newrelic_infra_alert_condition.go in this repo.  Using a local build while we wait on the PR.

**Comment by 0x91** [comment]
@smithclay is this provider still being maintained or is it abandonware?

**Comment by 0x91** [comment]
@mitchellh please can you let us know if this project has been abandoned??

**Comment by paultyng** [comment]
Closed in #48
